wwwroot/css/order-history.css

```
@@ -0,0 +1,203 @@
+}
+
+.lp-orders-wrap {
+  max-width: 1140px;
+}
+
+.lp-orders-head {
+  margin-bottom: 18px;
+}
+
+.lp-orders-head h1 {
+  font-family: "Work Sans", system-ui, -apple-system, Segoe UI, Roboto, Arial, sans-serif;
```

Stylelint (17.4.0)

[error] Expected quotes around "Segoe UI" (font-family-name-quotes)

(font-family-name-quotes)